third_party/3/orjson.pyi

```diff
@@ -3,9 +3,9 @@
 from typing import Any, Callable, Optional, Union
 
 def dumps(
-    obj: Any, default: Optional[Callable[[Any], Any]], option: Optional[int]
+    __obj: Any, default: Optional[Callable[[Any], Any]] = ..., option: Optional[int] = ...
 ) -> bytes: ...
-def loads(obj: Union[bytes, str]) -> Any: ...
+def loads(__obj: Union[bytes, str]) -> Union[dict, list, int, float, str, None]: ...
```

Comment:
We try to avoid Union return types, as mentioned in the CONTRIBUTING file, since they tend to be annoying to use for users.

Reply:
Oh, thanks, it's fixed now